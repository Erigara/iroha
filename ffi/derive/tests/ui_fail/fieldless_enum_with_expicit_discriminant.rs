use iroha_ffi::FfiType;

#[derive(FfiType)]
enum EnumWithExplicitDiscriminant { 
    A = 1,
    B,
    C,
    D,
}

fn main() {}
