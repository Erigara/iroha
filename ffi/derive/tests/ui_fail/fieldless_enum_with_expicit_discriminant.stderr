error: fieldless enums with explicit discriminants are prohibited
 --> tests/ui_fail/fieldless_enum_with_expicit_discriminant.rs:4:6
  |
4 | enum EnumWithExplicitDiscriminant {
  |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
